src/ol_infrastructure/applications/unified_ecommerce/__main__.py

```diff
@@ -1037,8 +809,8 @@
                 "plugin_config_name": shared_plugin_config_name,
                 "backends": [
                     {
-                        "serviceName": ecommerce_service_name,
-                        "servicePort": ecommerce_service_port_name,
+                        "serviceName": ol_k8s_application.ol_app_k8s_config.application_lb_service_name,  # noqa: E501
```

Comment:
This is good 👍  You've got the right ideas. 

Instead of referring to the config resource we should use the self values I mentioned in another comment.

Reply:
I'm sorry I'm really confused as to what this means.

Do you mean self in the context of the OLApplicationK8s class? Because the unified_ecommerce __main__.py doesn't have a self?

<- confused

I'm gonna hunt you down and make you ELI5 tomorrow :\

Reply:
Tobias explained it. Gotcha all set.